client/extensions/woocommerce/app/dashboard/manage-no-orders.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import BasicWidget from '../../components/basic-widget';
+import ExtensionsWidget from '../../components/extensions-widget';
+import ReadingWidget from '../../components/reading-widget';
+import ShareWidget from '../../components/share-widget';
+import WidgetGroup from '../../components/widget-group';
+
+class ManageNoOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
+		} ),
+	};
+
+	itemLink = ( path ) => {
+		const link = path.replace( ':site', this.props.site.slug );
+		return link;
+	}
+
+	renderShareWidget = () => {
+		const { translate } = this.props;
+		return (
+			<ShareWidget
+				text={ translate( 'Now that your setup is complete it\'s time to spread the word' +
+					' and get those first orders rolling in. Share a link with your friends,' +
+					' family & followers now.' ) }
+				title={ translate( 'Your store is ready & the world is waiting!' ) }
+				urlToShare={ this.itemLink( 'https://:site' ) }
+			/>
+		);
+	}
+
+	renderThemeWidget = () => {
+		const { translate } = this.props;
+		return (
+			<BasicWidget
+				className="dashboard__theme-widget"
+				buttonLabel={ translate( 'View themes' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'View Themes' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).